.github/workflows/Dockerfile.dev.yml

```diff
@@ -4,6 +4,10 @@ on:
   push:
     branches: master
   pull_request:
+  schedule:
```

Comment:
❓ does this mean the builds are triggered on pull request and on a schedule? do we need both?

Reply:
They'll run on the master branch on a schedule, and on pull requests. There will likely be large gaps of time between pull requests so I think it's worth running on a schedule.

Reply:
ah I see makes sense 👍